glama/echo-mcp.py

```
@@ -0,0 +1,40 @@
+        self.wfile.write(body)
+    def do_POST(self):
+        n = int(self.headers.get("Content-Length", 0))
+        req = json.loads(self.rfile.read(n) or b"{}")
+        rid, method = req.get("id"), req.get("method", "")
+        if method == "initialize":
+            self._json(200, {"jsonrpc": "2.0", "id": rid, "result": {
+                "protocolVersion": req.get("params", {}).get("protocolVersion", "2025-06-18"),
+                "capabilities": {"tools": {}},
+                "serverInfo": {"name": "echo-upstream", "version": "1.0.0"}}})
+        elif method == "notifications/initialized":
+            self.send_response(202); self.send_header("Content-Length", "0"); self.end_headers()
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)